counterpoll/main.py

```diff
@@ -559,6 +590,8 @@ def show():
     if wred_port_info:
         data.append(["WRED_ECN_PORT_STAT", wred_port_info.get("POLL_INTERVAL", DEFLT_1_SEC),
                     wred_port_info.get("FLEX_COUNTER_STATUS", DISABLE)])
+    if srv6_info:
+        data.append(["SRV6_STAT", srv6_info.get("POLL_INTERVAL", DEFLT_10_SEC), srv6_info.get("FLEX_COUNTER_STATUS", DISABLE)])
```

Comment:
Just want to confirm the 10-second default polling interval comes from arch spec, right?

Reply:
> Just want to confirm the 10-second default polling interval comes from arch spec, right?

need to dobule-check